doc/input-output-reference/src/overview/group-airflow.tex

```diff
@@ -1884,6 +1931,22 @@ \subsubsection{Outputs}\label{zoneearthtube-outputs}
 
 This is the humidity ratio of the air entering the zone after passing through the earth tube {[}kgWater/kgDryAir{]}.
 
+\paragraph{Earth Tube Node Temperature <X> {[}C{]}}\label{earth-tube-node-temperatures-c}
+
+This will generate the internal node temperatures {[}C{]} for the earth tube as a result of the 1-D finite difference model.  This is only valid for the 1-D (Vertical) model.
```

Comment:
This will **report** the internal node temperatures...

Right?  I don't think this will trigger the generation of anything.

Reply:
Yes, correct--it just triggers the reporting.  These always get calculated.  And before I forget, thanks for reviewing this!